portfolio/src/main/webapp/script.js

```diff
@@ -58,4 +58,13 @@ function createCommentElement(name, message, time) {
   const commentElement = document.createElement('li');
   commentElement.innerText = name + ' posted ' + message + ' on ' + time;
   return commentElement;
-}
\ No newline at end of file
+}
+
+// Creates a map and adds it to the page
+var map;
```

Comment:
Since the variable `map` is not being used anywhere, you can simply inline it with the `google.maps.Map` creation.